server/mongo_data_access.go

```diff
@@ -65,6 +118,11 @@ func (dal *mongoDataAccessLayer) Put(id string, resource interface{}) (createdNe
 	collection := dal.Database.C(models.PluralizeLowerResourceName(resourceType))
 	reflect.ValueOf(resource).Elem().FieldByName("Id").SetString(bsonID.Hex())
 	updateLastUpdatedDate(resource)
+
+	if dal.hasInterceptorsForVerbAndType("PUT", resourceType) {
```

Comment:
Same as for POST.  The `if` probably isn't necessary.  It's really only helpful if it can prevent the program from doing extra work that's only needed if there are interceptors (like in the DELETE case).